tests/testing/LocalBearTestHelperTest.py

```diff
@@ -25,6 +30,25 @@
 validTest = verify_local_bear(TestBear,
                               valid_files=files,
                               invalid_files=())
+min_files = ('This is valid.',)
+max_files = ('This is particularly an invalid file',)
+AspectTest = verify_local_bear(AspectsGeneralTestBear,
+                               valid_files=min_files,
+                               invalid_files=max_files,
+                               aspects=AspectList([
+                                    get_aspect('LineLength')(
+                                        'Unknown', max_line_length=20),
+                                    ]),
+                               )
+PriorityAspectsTest = verify_local_bear(AspectsGeneralTestBear,
```

Comment:
finish this line at the open `(`, and dedent everything after it to start at 4 spaces and keep indenting from there.

then you wont hit 80cpl so quickly and need to squish everything in the last 10 chars.